test/js/api/filters-spec.js

```diff
@@ -185,4 +185,38 @@ describe('api/filters', function() {
 
         assert.equal(expected, filterCss(['css.all']));
     });
+
+    it('should escape special characters', function() {
+        const actual = 'stra$nge?name*for.a.fi*le',
+            expected = 'stra\\$nge\\?name\\*for\\.a\\.fi\\*le';
+        assert.equal(expected, filters.apply(actual, ['escape']));
+    });
+
+    //it('TODO: regexp - how is it used?', function() {});
```

Comment:
We've discuss this, just adding it here for visibility

regexp filter reverts the escape filter on the text between two '&rx;'. This allows us to use regular expressions with xml files that contain characters that need to be escaped